Example/PaymentSheet Example/PaymentSheet Example/CustomerSheetTestPlayground.swift

```diff
@@ -53,6 +53,9 @@ struct CustomerSheetTestPlayground: View {
                                 SettingPickerView(setting: $playgroundController.settings.paymentMethodRemove)
                                 SettingPickerView(setting: $playgroundController.settings.paymentMethodRemoveLast)
                                 SettingPickerView(setting: $playgroundController.settings.paymentMethodAllowRedisplayFilters)
+                                if playgroundController.settings.alternateUpdatePaymentMethodNavigation == .on {
```

Comment:
When can we remove alternateUpdatePaymentMethodNavigation?

Reply:
I suppose if we're trying to release this is the default flow, it should be removed now

Reply:
I do anticipate possibly having to update a lot of tests though, so might be changing a lot of files